homeassistant/components/roborock/select.py

```diff
@@ -79,6 +79,10 @@ async def async_setup_entry(
         )
         is not None
     )
+    async_add_entities(
+        RoborockCurrentMapSelectEntity(f"selected_map_{coordinator.duid}", coordinator)
```

Comment:
Don't you need `slugify(coordinator.duid)` here, you use it everywhere else?

Reply:
Yes you are right! Next time I'm back at my computer I'll make the change. It's unfortunately not too easy to do on my phone

Reply:
Added!